creator-node/scripts/start.sh

```diff
@@ -71,9 +71,9 @@ if [[ "$devMode" == "true" ]]; then
         npm link
         cd ../app
         npm link @audius/sdk
-        npx nodemon --exec 'node --inspect=0.0.0.0:${debuggerPort} --require ts-node/register src/index.ts' --watch src/ --watch ../audius-libs/ | tee >(logger) | ./node_modules/.bin/bunyan
+        npx nodemon --exec 'node --inspect=0.0.0.0:${debuggerPort} --require ts-node/register src/index.ts' --watch src/ --watch ../audius-libs/ | tee >(logger)
```

Comment:
does this affect how loggly / log formatting for local logs works?

Reply:
It does not affect loggly since line 68 is hidden behind a logical gate that checks if we're in `devMode`.

It does affect local logs, but the workaround can be found within the description of this PR under the `Semi-Breaking change for Content Node Logs` heading.